troi/cli.py

```diff
@@ -20,8 +20,12 @@ def cli():
 ))
 @click.argument('patch', type=str)
 @click.option('--debug/--no-debug')
+@click.option('--print', '-p', 'echo', required=False, is_flag=True)
+@click.option('--save', '-s', required=False, is_flag=True)
+@click.option('--token', '-t', required=False, type=click.UUID)
```

Comment:
I would like to see documentation for these flags. See how I did it for the other patches in my previous PR - click doesn't let add documentation directly to options, so you have to do it by writing a docstring.